### DISCUSSION TITLE
Is there an equivalent to PyInquirer to make a prompt with multiple options?

### DISCUSSION THREAD

**Discussion by kevgilmore** [remkop/picocli#1974] [Q&A]
As seen in https://github.com/CITGuru/PyInquirer

**Comment by remkop** [comment]
Hi @kevgilmore!
Picocli at its core is a command line parser.
However, from the model it can also generate command line completion scripts for various shells (bash, zsh, and jline).

The [jline2 shell](https://github.com/remkop/picocli/tree/main/picocli-shell-jline2) and [jline3 shell](https://github.com/remkop/picocli/tree/main/picocli-shell-jline3) subprojects offer some building blocks, but the effort has mostly been on command line autocompletion. It is very rudimentary, nothing as sophisticated as PyInquirer.

If you are interested in contributing to give picocli more capabilities in the direction of PyInquirer, that would be great!
